jadx-gui/src/main/java/jadx/gui/ui/TabbedPane.java

```diff
@@ -11,16 +11,13 @@
 import java.util.List;
 import java.util.Map;
 
+import jadx.gui.treemodel.*;
```

Comment:
I suggest using implicit imports (don't use `.*`)

Reply:
Right. IntelliJ seem to have a strange understanding of optimized import :(